can.py

```diff
@@ -228,26 +251,167 @@ def export_c_library(self, filename: str = "can_ids.h"):
         self.export_h("can_ids.h")
         self.export_h("can_parser_types.h")
 
+    def export_csv(self, filename: str):
+        import pandas as pd
+        
+        modules = []
+        signature = []
+        ids = []
+        names = []
+        frequency = []
+        load = []
+        frame_length = []
+        description = []
+
+        for module in self.modules:
+            for topic in module["topics"]:
+                modules.append(module["name"])
+                signature.append(module["signature"])
+                ids.append(topic["id"])
+                names.append(topic["name"])
+                frequency.append(round(topic["frequency"],3))
+                load.append(round(self.get_topic_load(topic),3))
+                frame_length.append(topic["frame_length"])
+                description.append(topic["description"])
+
+        df = pd.DataFrame({
+            "modules":modules,
+            "signature": signature,
+            "name": names,
+            "ids": ids,
+            "frequency": frequency,
+            "load": load,
+            "frame_length": frame_length,
+            "description": description,
+            })
```

Comment:
Since this data is small it is okay doing it this way, but note that for bigger data, possibly it would be better convert it to a DataFrame first, then group / add / remove columns.